test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -1831,6 +1831,8 @@ TEST_F(AtenXlaTensorTest, TestArgMin) {
     torch::Tensor xla_b = torch::argmin(xla_a, c10::nullopt, /*keepdim=*/false);
     AllEqual(b, xla_b);
   });
+
+  ExpectCounterNotChanged("aten::.*", cpp_test::GetIgnoredCounters());
```

Comment:
I think we need to start asserting for things we want to see?

https://github.com/pytorch/xla/pull/1250

Like:
```Python
ExpectCounterNotChanged("aten::.*", cpp_test::GetIgnoredCounters());
ExpectCounterChanged("xla::...", cpp_test::GetIgnoredCounters());
```

Reply:
Hmmm I'm afraid adding positive assert for non leaves are too much. I agree it's good to add for leaves. This is a moving part in Pytorch, I feel asserting that will force us to change tests very frequently?

Reply:
Nvm, let's add them now and delete in the future if too much noise ;) 
Sometimes it's good to get a signal when the lowered graph changes as well. Hopefully it doesn't happen often.

Reply:
Yes. The point is, if today we assume pytorch does a certain lowering, and tomorrow it changes to a different one which is less performant, we better know.